cypress/integration/mocked-okta/registerController.1.cy.ts

```diff
@@ -0,0 +1,91 @@
+import userStatuses from '../../support/okta/userStatuses';
+import userResponse from '../../fixtures/okta-responses/success/user.json';
+import userExistsError from '../../fixtures/okta-responses/error/user-exists.json';
+import successTokenResponse from '../../fixtures/okta-responses/success/token.json';
+import resetPasswordResponse from '../../fixtures/okta-responses/success/reset-password.json';
+
+beforeEach(() => {
+  cy.mockPurge();
+});
+const verifyInRegularEmailSentPage = () => {
+  cy.contains('Check your email inbox');
+  cy.contains('Resend email');
+};
+
+userStatuses.forEach((status) => {
+  context(`Given I am a ${status || 'nonexistent'} user`, () => {
+    context('When I submit the form on /register', () => {
+      beforeEach(() => {
+        cy.visit(`/register?useOkta=true`);
+        cy.get('input[name="email"]').type('example@example.com');
+      });
+      switch (status) {
+        case false:
+          specify(
+            "Then I should be shown the 'Check your email inbox' page",
+            () => {
+              // Set the correct user status on the response
+              const response = { ...userResponse.response, status };
+              cy.mockNext(userResponse.code, response);
+              cy.get('button[type=submit]').click();
+              verifyInRegularEmailSentPage();
+            },
+          );
+          break;
+        case 'ACTIVE':
```

Comment:
I don't know what we would do here to test register with social users ? @coldlink  do we need to mock pushing the apple, facebook and google button ?